about/triage-meeting.md

```diff
@@ -12,9 +12,9 @@ The triage meeting takes place on [Zulip](chat-platform.md). Look for a topic li
 ## Working group check-in
 This section contains the scheduled check-ins for working groups:
 
-- **2019-03-14:** [wg-async-await], [wg-llvm]
 - **2019-03-21:** [wg-nll], [wg-traits]
 - **2019-03-28:** [wg-parallel-rustc], [wg-pgo]
+- **2019-04-24:** [wg-rls-2.0], [wg-meta]
```

Comment:
Maybe we do want to unify these into one list, though, so that no update is needed?

Reply:
I'm not sure I understand what you mean. 

I spent a little bit of time trying to think of a nice way to encode the list (or refer to the row number of the big table in the README) and then work out what it should be for this coming meeting, but I couldn't think of anything that was sufficiently easy.

Reply:
Oh, I just meant having a list by date, and assuming people could tell which dates were in the future, and which were in the past.